db/migrate/20220727083528_res05_foodvalley/data.csv

```diff
@@ -0,0 +1,2 @@
+RES05,foodvalley,nl,315.583398,307.744154,11.673248,0,0,0,0,0,0,0,0,0,1,0.356447,0.032287,0.486212,0.125054,0.001659,0.998341,0,1,0,0,1,0,1,0,0,1,1,0,0,0,0.012889,0.955206,0.031905,0.978823,0,0.021177,220.64,0,2284.136174,2663.25255,161,382,0,0,0,0,0,0,0,0,0,1,12477,6.994403,0.46,49,0,0,0,0,0,81.569951,0,0,0,0.137104,0,0.862896,0,1,0,0,0,0,169560,250985836.7,5068.503687,5078.890552,118.274501,25.156241,350.591013,304.408987,0.505602,0.03913,0,0,0.96,0.01,0,0,0.03,0.22,0.07,0.03,0.68,0.75,0.1,0,0.08,0.07,0.00E+00,1,0,0,0,1,0,0,0,0,1,6.044676,-5.044676,0.0565,0.0733,0.0945,0.1012,0.1052,0.0743,0.072,0.0774,0.0715,0.0565,0.0667,0.102778,0.03597,2.511443,36.8368,279.64952,0.008244,0.001288,0.009731,0.00469,0.000645,0.028939,0.002595,0,0.010102,0.000591,0.000551,0.008729,0.000706,0.410722,0.02925,0.197382,0.05211,0.317468,0,687.77,322.65,30060,0.215,2019,13730961.47,11800045.01,6934135.542,4891655.023,4119288.441,0,0,18933586.17,0.25,0.25,0.25,0.25,0.25,480000,0.02,18320,0.2,138000,0.2,72800,0.02,4000,0.2,50000,5.220074,6.1142,6.943599,8.364381,11.473256,4.538455,5.915226,7.019672,8.409771,11.404574,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2069.333333,0,0,867,0,0,0,0,0,1361.3,0,3200,0,0,4530,0,0,0,0,0,0,8500,0,6000,0,0,0,0,2190,2190,2190,2190,2190,2190,2190,0,0,6500,0,0,0,0,0,2.5488,371.491591,67.90862,0,0,36,46,36.1,42,45.3,37.3,40,30.5,40,15,43,14,40,15,35,15,34,60,49,40,48,40,50,43.3,30.5,24.75,35,43,47,43.3,30.5,45,38,25,27,36.93,36.93,41,40,18.3,47,20.45,19.95,18.3,47,80,72,109.593,92.4,91.5,72,72,72,92.4,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,9.338598,0,0,0,0,268,0,0,0,0,77,0,0,0,0,0,39,0,0,0,0,0,0,0,0,25.625,0,0,0,0,0,1,0.58,0.42,1806.94087,1398.308474,216.367101,0,0,0,0,0,0,0,356123,147317,7.685466,0.297896,0.301415,0.142087,0.104299,0.154303,0.188107,0.21,0.21,0.114227,0.115425,0.15,183.36,0,0.034247,0.011493,0.032963,0.104645,0.072882,0.74377,0.020498,0.15188,0.827622,0,0.155058,0.844942,0,0.983146,0.016854,0,1,0,1,0.963886,0.035793,0.000321,0.675787,0.238596,0.01131,0.074307,0.71374,0.188931,0.097329,0.791874,0.187673,0.020453,0.098361,0.131148,0.081967,0.295082,0.114754,0.131148,0.065574,0.081966,0.722722,0.020224,0.257054,1524.214236,5437.47,214,356,0,0,0,0,0,0,0,0,0,1,0.00015,0.8,0.2,7.0548,0.61,0.981459,0.018541,0.8,0.2,0,0.16,0.11,0.73,0,0,1,0,0,1,0.77,0.23,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,1,690.181988,795.886772,0,0,0,0,1,0,774.233727,787.846158,0,0,0,0,0,0,0,0,0,0,169.2684,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,67.825182,66.66378,0,0,0,0,0,0,0,0,33.244804,39.72714,0,0,0,0,1,0
```

Comment:
The name of foodvalley is not written with a capital letter like we have for the other areas.